etc/adminhtml/system.xml

```diff
@@ -227,6 +227,12 @@
                         <config_path>payment/amwal_payments/sentry_report</config_path>
                         <tooltip>Enable reporting of errors to Amwal Sentry</tooltip>
                     </field>
+                    <field id="quote_override" translate="label tooltip" type="select" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
+                        <label>Quote override</label>
+                        <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                        <config_path>payment/amwal_payments/quote_override</config_path>
+                        <tooltip>Enable quote override</tooltip>
```

Comment:
It might be good to make this a bit more descriptive
```suggestion
                        <tooltip>If enabled, will override certain quote data. Use only if you are having issues related to quote amounts or currencies</tooltip>
```